concerned-coyotes/earlyinternet/commands/management/commands/run_tasks.py

```diff
@@ -0,0 +1,24 @@
+import importlib
+import time
+
+from django.core.management.base import BaseCommand
+from django_q.models import Schedule
+
+
+class Command(BaseCommand):
+    help = "Run all scheduled tasks"
+
+    def handle(self, *args, **options):
+        tasks = Schedule.objects.all()
+
+        for i, task in enumerate(tasks, start=1):
+            task: Schedule
+
+            module, func_name = task.func.rsplit(".", 1)
+            module_obj = importlib.import_module(module)
+            func = getattr(module_obj, func_name)
+            print(f"Now running {task.func} {i}/{len(tasks)}")
```

Comment:
In the `setup_tasks.py` file, another `BaseCommand` subclass uses the `self.stdout.write` method to write output to stdout instead of using `print` directly. This `self.stdout.write` approach is the recommended approach in the Django documentation. It would probably have been better to use it here as well even if it's just for consistency. (As you're running tests, not printing directly could be a benefit there as well.)